shell/common/platform_util_linux.cc

```diff
@@ -408,15 +410,7 @@ bool PlatformTrashItem(const base::FilePath& full_path, std::string* error) {
 }  // namespace internal
 
 void Beep() {
-  // echo '\a' > /dev/console
-  FILE* fp = fopen("/dev/console", "a");
```

Comment:
Was there a reason they didn't use `gdk_display_beep`? I'm 👍 on the change as long as there's not some kind of historical context we might not have!

Reply:
I went back through the history of our implementation and didn't see any discussion of it, either pro or con, so I can only guess.

`gdk_display_beep()` existed when our `Beep()` was written, but it's possible that the person who wrote it didn't know about it.